source/includes/dev/_microservice_dev_guide.md

```diff
@@ -195,6 +200,88 @@ Running tests under Docker is similar to running the actual microservice. The ea
+- **Your microservice crashes when handling an API call, trying to reach a network address**: this may be your microservice trying to reach another microservice through Control Tower. Make sure that both Control Tower and the necessary dependent microservices are up and running, and that all microservices involved are registered in Control Tower. Be sure that Control Tower's cron is running.
+- **Your microservice has user-related issues, even though you are providing a `Bearer` token**: Bearer tokens are processed by Control Tower, and transformed into a `loggedUser` JSON object that microservices expect. Either provide said object directly to your microservice, or route you request with the token through Control Tower.
+- **Your tests keep failing**: This can be due to multiple reasons. Check the microservice's travis status (link in the README.md) to see if it's just you, or if there's an issue with the preexisting code base. Run your tests a few more times and see if the output is consistent - some tests are not deterministic, and have varying results.
+
+## CI/CD
+
+The RW API uses multiple tools in it's [CI](https://en.wikipedia.org/wiki/Continuous_integration) and [CD](https://en.wikipedia.org/wiki/Continuous_delivery) pipelines. All microservices that compose the RW API use a common set of tools:
+
+- [Github](https://github.com) for version control and code repository.
+- [Travis CI](travis-ci.org/) for automatic test execution.
+- [Code Climate](codeclimate.com/) for code coverage monitoring.
+- [Jenkins](https://www.jenkins.io/) for deployment.
+
+We assume, at this point, that you're already familiar with Github and its core functionality, like branches and pull requests (PRs). If that's not the case, use your favourite search engine to learn more about those concepts.
+
+Each microservice lives in a separate Github repository, most of which have Travis and Code Climate integrations configured. Whenever a pull request is created, both tools will be triggered automatically - Travis will run the tests included in the code, and notify the PR author of the result. Code Climate builds on top of that, and monitors and reports [code coverage](https://en.wikipedia.org/wiki/Code_coverage). The behavior of both tools is controlled by a single `.travis.yml` file you'll find in the root of each microservice's code base, and you can learn about it on each of the tool's documentation page. You can see the result directly on the PR page.
+
+When you want to submit a change to the code of one of the microservices, you should:
+
+- Do your changes in a separate git branch, named after the change you're making.
+- Target the `dev` branch (or `develop`, if `dev` does not exist yet).
```

Comment:
Would consider adding a little info here - we are in the process of renaming branches, etc.

Reply:
IMO that's a temporary process we're carrying out that, by the time anyone reads these docs, will be mostly done.